src/utils/filterResults.ts

```diff
@@ -7,13 +7,13 @@ type MatchTallies = {
   propertyGroup: number
 }
 
-function matchForPropertyTypes (property: UnprocessedResultsFromCRM, desiredPropertyTypes: string[], maxPropertyTypes: boolean): boolean {
+function matchForPropertyTypes (property: UnprocessedResultsFromCRM, desiredPropertyTypes: string[], maxPropertyTypes: boolean | undefined): boolean {
```

Comment:
Below comment is mostly unrelated to work in this commit and I'm happy to do this myself in a separate commit:
I don't think `maxPropertyTypes` needs to be passed to this function. 

It could be handled in this statement:
```const propertyTypeMatch = matchForPropertyTypes(property, desiredPropertyTypes, maxPropertyTypes)```

I would do:

1. Rename the variable maxPropertyTypes: it is misleading (makes me think it's the maximum number of property type matches). I'd call it `isUnderPropertyTypeLimit`

2. Change to ```const propertyTypeMatch = isUnderPropertyTypeLimit && matchForPropertyTypes(property, desiredPropertyTypes)```

Same applies to property groups as well.

Reply:
No worries. I'm happy for you to do it in a separate commit as well :) lol